packages/components/inputs/localized-rich-text-input/src/editor.js

```diff
@@ -75,7 +81,18 @@ const Editor = (props) => {
     onToggle();
   }
 
-  const languagesControl = props.languagesControl();
+  const shouldToggleButtonTakeSpace =
+    /*
+      - if hasLanguagesControl and there are no errors/warnings to display
+      - then the toggleButton is absolutely positioned
+      This is because the toggle button is placed next to the LocalizedInputToggle without being siblings in the DOM.
+      If there is a error or warning showing,
+      then it can be placed statically because it will then be a sibling to the error/warning message
+      and LocalizedInputToggle is placed below the errors/warnings.
```

Comment:
I'm wondering if it is worth mentioning in a README about this.
so that there won't be any surprise

Reply:
Hmm, I don't think it's necessary, because this isn't relevant to consumers of the package.

The positioning nuance is all dealt with internally, and the comment is just explaining how it's being done, for the sake of maintainability.